moments.yaml

```diff
@@ -47,3 +28,23 @@
   template: update_action
   trigger:
     id: momentsUpdate
+- id: WNP_MOMENTS_7
+  groups: [moments-pages]
+  content:
+    action:
+      data:
+        expireDelta: 604800000
+        url: https://www.mozilla.org/firefox/welcome/7/
+      id: moments-wnp
+    bucket_id: WNP_MOMENTS_7
+  # Fx 70+, profile age > 30 days all en-* and selected locales
+  targeting: firefoxVersion >= 69 &&
+    (
+      localeLanguageCode == "en" ||
+      locale in ["be", "cy", "da", "de", "dsb", "es-AR", "es-CL", "es-ES", "es-MX", "fr", "fy-NL", "gn", "hr", "hsb", "hu", "hy-AM", "ia", "id", "it", "ka", "kab", "nl", "nn-NO", "pl", "pt-BR", "pt-PT", "ro", "ru", "sk", "sl", "sq", "sr", "sv-SE", "tr", "uk", "vi", "zh-TW"]
+    ) && ((currentDate|date - profileAgeCreated) / 86400000) > 30
+    &&  xpinstallEnabled &&
+    (['@contain-facebook','{bb1b80be-e6b3-40a1-9b6e-9d4073343f0b}', '{a50d61ca-d27b-437a-8b52-5fd801a0a88b}'] intersect addonsInfo.addons|keys)|length == 0
```

Comment:
I'm not sure what those other 2 addons are, I could only confirm `Facebook Container` has the `@contain-facebook` id.

Reply:
Nvm: we should keep that list, it looks good. I found the trick `addons.mozilla.org/firefox/addon/<guid>`

Reply:
Yeah, I wasn't sure either at first. But looks like all the Facebook Container CFRs have those two included.